ncm-network/src/main/perl/network.pm

```diff
@@ -303,6 +303,31 @@ sub Configure {
         ### set the networktype
         if ( $net{$iface}{'type'} ) {
             $text .= "TYPE=".$net{$iface}{'type'}."\n";
+            ### Set OVS related variables
+            if ($net{$iface}{'type'} =~ /^OVS/) {
+                $text .= "DEVICETYPE='ovs'\n";
+                if ($net{$iface}{'ovs_bridge'}) {
+                    $text .= "OVS_BRIDGE='$net{$iface}{ovs_bridge}'\n";
+                }
+                if ($net{$iface}{'ovs_opts'}) {
+                    $text .= "OVS_OPTIONS='$net{$iface}{ovs_opts}'\n";
+                }
+                if ($net{$iface}{'ovs_extra'}) {
+                    $text .= "OVS_EXTRA='$net{$iface}{ovs_extra}'\n";
+                }
+                if ($net{$iface}{'bond_ifaces'}) {
+                    $text .= "BOND_IFACES=".join(' ',$net{$iface}{bond_ifaces})."\n";
```

Comment:
these so-called config files are sourced in a bash environment, so proper quoting around whitespace is required. there's nothing worng with single quotes, but there are misisng here :smile: 
fixed in https://github.com/quattor/configuration-modules-core/pull/461/files#diff-90bc6b2e2a105ae5c62daf55ef95b2a7R283